TodoApp/TodoApp.Api/Global.asax.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Unused using statements are generally frowned upon. You should get rid of them when you get a chance - but what's more important, keep them in mind when writing your own code.